docs/package.json

```diff
@@ -49,6 +49,7 @@
     "cross-env": "^7.0.3",
     "docsearch.js": "^2.6.3",
     "doctrine": "^3.0.0",
+    "exceljs": "^4.3.0",
```

Comment:
It's working on our documentation but I couldn't use the feature on codesandbox because it's missing this dependency.

Reply:
I don't know how to add packages to the codesandbox examples. But I'm not sure it is a good idea to add it for every user if it's needed for only one feature.
Adding a package on codesandbox is fast. Just write the name of the package in the dependencies field

![image](https://user-images.githubusercontent.com/45398769/157473779-b478f2dc-4db9-4d52-ac34-6bbd3cce9b4e.png)

Reply:
> I don't know how to add packages to the codesandbox examples

You are blocked by the freeze ?

Reply:
Easier than that, I don't know what is "the freeze", and I do not know how codesandbox are generated by the doc 😅

Reply:
Oh I thought you were talking about regular codesandboxes, not the one of the doc.

https://github.com/mui/mui-x/pull/3396/files

You have an example here about how to modify the deps injected in the codesandbox.
We just have to add `exceljs` for `@mui/x-data-grid-pro` I think.

Reply:
We can not add dependencies for specific examples. It works thanks to a regExp finding all the `import ... from ...` to know which are the dependencies.

As suggested, I added it as a pro dependency, does not seems to decrease perf of opening codesandbox 👍